cmake/modules/kconfig.cmake

```diff
@@ -297,10 +297,14 @@ if(EXISTS ${DOTCONFIG} AND EXISTS ${merge_config_files_checksum_file})
 endif()
```

Comment:
Typos in commit:
`This commit introducuced an internal CONFIG_FORCED CMake setting which` -> `This commit introduces an internal CONFIG_FORCED CMake setting which`
`allows higher order build systems to generated a configuration file` -> `allows higher order build systems to generate a configuration file`
`which will always take precendence` -> `which will always take precedence`

Reply:
done